Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -61,40 +59,82 @@ def test_cohesity_restore_from_last_snapshot(self):
         playbook_name = "Cohesity_Restore_From_Last_Snapshot"
         subscription_id = get_subscription_id()
         result = search_alert_id_in_incident(
-            self.alert_id, self.resource_group, self.workspace_name)
+            self.alert_id, self.resource_group, self.workspace_name
+        )
         jsObj = result[0]
+
+        # The expected format of jsObj["id"]:
+        # /subscriptions/<subscription_id>/resourceGroups/<resource_group>/
+        # providers/Microsoft.OperationalInsights/workspaces/<workspace>/
+        # providers/Microsoft.SecurityInsights/Incidents/<incident_id>
+        # Example:
+        # /subscriptions/26c46499-1b17-43a7-8d18-024c79f09bc7/resourceGroups/
+        # kishan-rg-3/providers/Microsoft.OperationalInsights/workspaces/
+        # kishan-rg3-ws4/providers/ Microsoft.SecurityInsights/Incidents/
+        # d7c70079-72e3-4122-a192-27944276c713
+
         incident_id = jsObj["id"].split("/")[-1]
 
         alert_details = get_alert_details(self.alert_id, self.api_key)
         alert = Alert(json.dumps(alert_details))
         protection_group_id = alert.get_protection_group_id()
         cluster_id = alert.get_cluster_id()
-        current_time_usecs = int(time.time() * 1000000)
-        start_time_usecs = current_time_usecs - 180000000
-        recoveries = get_recoveries(cluster_id, self.api_key,
-                                    str(start_time_usecs))
 
-        assert recoveries is None or not recoveries.get('recoveries') \
-            or not any(any(obj['protectionGroupId'] == str(protection_group_id)
-                       for obj in recovery['vmwareParams']['objects'])
-                       for recovery in recoveries['recoveries'])
+        # Constants for better readability
+        MICROSECS_IN_SEC = 1000000
```

Comment:
This is a generic macro which can be defined at the top.
THREE_MINUTES_IN_USECS should be a internal variable like

query_range_usecs = 3 * 60 * MICROSECS_IN_SEC

Reply:
done